test/unit/test-amp-story-player.js

```diff
@@ -370,5 +370,47 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
         'Story URL not found in the player: https://example.com/story6.html'
       );
     });
+
+    it('navigate to stories given a number', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      win.requestAnimationFrame(() => {
+        const iframes = playerEl.shadowRoot.querySelectorAll('iframe');
+        
+        player.go(0);
+        expect(iframes[0].getAttribute('i-amphtml-iframe-position')).to.eql('0');
+        player.go(1);
+        expect(iframes[1].getAttribute('i-amphtml-iframe-position')).to.eql('0');
+        player.go(-1);
+        expect(iframes[0].getAttribute('i-amphtml-iframe-position')).to.eql('0');
```

Comment:
Please split these in three tests, so when things fail it's easier to know if the error is when navigating backward or forward. It's good unit testing practice to only assert one thing per test.